examples/tutorial/lessons/tutorial-data.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Generate the following metrics at 5 second intervals
+#   Gauge   gauge-1   random values between 0 and 100
+#   Gauge   gauge-2   random values between 0 and 100
+#   Counter counter-1 increasing counter, incremented 0..5 each time
+
```

Comment:
A common issue I am seeing in testing environments is when people send data to metric to fast of the <1 seconds interval.
I think in lesson 01 it could deserves a couple of lines explaining it or even adding the variable HAWKULAR_MIN_REPORTING_INTERVAL_DATA commented as optional if user wants to modify this script and results are not coming.
I think it might save time of debugging/troubleshooting.